modules/ossm-federation-config-smcp.adoc

```diff
@@ -0,0 +1,397 @@
+////
+This module included in the following assemblies:
+* service_mesh/v2x/ossm-federation.adoc
+////
+
+[id="ossm-federation-config-smcp_{context}"]
+= Configuring a control plane for federation
+
+Before a mesh can be federated, you must configure the `ServiceMeshControlPlane` for mesh federation.   Because  all meshes that are members of the federation are equal, and each mesh is managed independently,  you must configure the SMCP for _each_ mesh that will participate in the federation.
```

Comment:
```suggestion
Before a mesh can be federated, you must configure the `ServiceMeshControlPlane` for mesh federation. Because  all meshes that are members of the federation are equal, and each mesh is managed independently, you must configure the SMCP for _each_ mesh that will participate in the federation.
```
Surplus spaces?